templates/miq-template.yaml

```diff
@@ -158,6 +220,9 @@ objects:
             -
               name: "MEMCACHED_SERVICE_NAME"
               value: "${MEMCACHED_SERVICE_NAME}"
+            -
+              name: "BINDING_ADDRESS"
```

Comment:
If not specified, the rails servers listen on 127.0.0.1 and don't listen for requests on any other IP.  We need to tell it to listen on all interfaces to respond to the requests forwarded from the httpd pod.